crates/glaredb/src/local.rs

```diff
@@ -160,6 +161,25 @@ impl LocalSession {
                             Ok(_) => {}
                             Err(e) => println!("Error: {e}"),
                         };
+
+                        // Print out notices as needed.
+                        //
+                        // Note this isn't being called in the above `execute`
+                        // function since that can be called in a
+                        // non-interactive fashion which and having notice
+                        // messages interspersed with the output would be
+                        // annoying.
+                        for notice in self.sess.drain_notices() {
+                            println!(
```

Comment:
do we want to write to stderr?

Reply:
We could. Since this is in the interactive portion, I don't know that it matters. But we could also move this to the `execute` func and eprintln there.

Reply:
stderr feels right for this kind of thing, (if only because it makes it easier to write scripts.) but this doesn't need to block.

Reply:
updated